src/python/TaskWorker/Actions/DagmanCreator.py

```diff
@@ -366,12 +366,19 @@ def __init__(self, config, crabserver, procnum=-1, rucioClient=None):
     def populateGlideinMatching(self, info):
         scram_arch = info['tm_job_arch']
         # Set defaults
-        info['desired_arch'] = "X86_64"
+        info['required_arch'] = "X86_64"
+        # The following regex matches a scram arch into four groups 
```

Comment:
THANKS ! this explanation was dearly needed

Reply:
:)